Backend/users/usersModels.js

```diff
@@ -0,0 +1,48 @@
+const db = require("../data/dbConfig");
+
+const findBy = filter => {
+  return db("users")
+    .where(filter)
+    .first();
+};
+
+const findById = id => {
+  return db("users")
+    .where({ id })
+    .select(
+      "id",
+      "username",
+      "email",
+      "push_notification"
+    )
+    .first();
+};
+
+const findByUsername = username => {
+  return db("users")
+    .where({ username })
+};
+
+const createUser = async user => {
+  return db("users").insert(user, ["id", "email", "username"]);
```

Comment:
You still do not show what fields are returning from database

Reply:
I did you will  see that ["id", "email", "username"] indicates that after inserting the user into the "users" table, the function will return the id, email, and username fields of the newly inserted user